validation/telemetry/main.4.hdd_type_null.fail.json

```diff
@@ -0,0 +1,43 @@
+{
+  "application": {
+    "architecture": "x86-64",
+    "buildId": "20151103030248",
+    "channel": "beta",
+    "name": "Firefox",
+    "platformVersion": "45.0",
+    "vendor": "Mozilla",
+    "version": "45.0",
+    "xpcomAbi": "x86_64-gcc3"
+  },
+  "creationDate": "2015-11-05T01:25:43.312Z",
+  "type": "main",
+  "id": "0fdac909-d2ec-454c-b625-261a3e5d5c9b",
+  "version": 4,
+  "environment": {
+    "build": {
+      "applicationId": "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}",
+      "applicationName": "Firefox",
+      "architecture": "x86-64",
+      "buildId": "20180509105613",
+      "version": "62.0a1",
+      "vendor": "Mozilla",
+      "platformVersion": "62.0a1",
+      "xpcomAbi": "x86_64-gcc3"
+    },
+    "partner": {},
+    "settings": {},
+    "system": {
+      "hdd": {
+        "profile": {
+          "type": null
```

Comment:
I actually didn't know this would fail. It looks like there needs to be a slight modification for this to work properly.

Reply:
Ah yep, `null` should pass. It is fixed now